src/main/java/com/twilio/taskrouter/application/servlet/EventsServlet.java

```diff
@@ -25,48 +31,85 @@
 @Singleton
 public class EventsServlet extends HttpServlet {
 
-  public static final String LEAVE_MSG = "Sorry, All agents are busy. Please leave a message. "
+  protected static final String LEAVE_MSG = "Sorry, All agents are busy. Please leave a message. "
```

Comment:
Well this is one of the advantage of using constant variables because the text is actually relevant, we need to keep the referential integrity. But i can change it.